toolkit/tools/imagecustomizerapi/verity.go

```diff
@@ -8,8 +8,9 @@ import (
 )
 
 type Verity struct {
-	DataPartition IdentifiedPartition `yaml:"dataPartition"`
-	HashPartition IdentifiedPartition `yaml:"hashPartition"`
+	DataPartition    IdentifiedPartition `yaml:"dataPartition"`
+	HashPartition    IdentifiedPartition `yaml:"hashPartition"`
+	CorruptionOption *CorruptionOption   `yaml:"corruptionOption"`
```

Comment:
Since `CorruptionOption` has a default value (`""`), you don't need to make this a pointer.

Reply:
I make it pointer because sometimes user may want to skip giving value for `corruptionOption`, but still this arg should have a default value.

For example:

```
verity:
  dataPartition:
    idType: part-label
    id: root
  hashPartition:
    idType: part-label
    id: root-hash
```

Even though user did not provide arg `corruptionOption`, but under the hood it has been assigned `""`.

Reply:
wouldn't it under the hoold have `io-error` in this case?

Reply:
> wouldn't it under the hood have io-error in this case?

Correct, the previous comment is out of date.

Reply:
Unmarshal the yaml with zero of type to look for the default arg prepared. Ok, got it. Will revise.